app/models/invitation.rb

```diff
@@ -0,0 +1,4 @@
+class Invitation < ApplicationRecord
+  belongs_to :attendee, class_name: 'User'
+  belongs_to :attended_event, class_name: 'Event'
```

Comment:
- **[OPTIONAL]** Let's add uniqueness validation on [attendee_id, attended_even_id] combination, to make sure that a user can attend the same event only once. Take a look at [scope validation](https://guides.rubyonrails.org/active_record_validations.html#uniqueness) to specifiy multiple attributes on validations.